packages/ethereum/README.md

```diff
@@ -58,6 +58,14 @@ npx hardhat coverage
 
 > tip: see coverage results by launching `./coverage/index.html`
 
+# Deploy
+
+To deploy locally, run
+
+```bash
+yarn prepack && yarn deploy:local
```

Comment:
Why is `yarn prepack` required?

Reply:
It happens to have the command that should be used in the local deployment. I made the commands in the readme explicit now